docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   suave-mevm:
-    image: flashbots/suave-geth:latest
+    image: flashbots/suave-geth:v0.1.2
```

Comment:
I am hesitant to pin a specific version because it will require us to open a PR to suapp-examples each time a new change to suave-geth is released.